### DISCUSSION TITLE
Serving documentation without building

### DISCUSSION THREAD

**Discussion by nlosacco** [idaholab/moose#26344] [Q&A Tools]
Hi all,

I like to serve the documentation on my laptop, it’s quite useful since I travel a lot and I’m often offline.  How can I start the documentation server without having to re-build the documentation each and every time? So far I used:
`./moosedocs.py build --serve`

thanks

**Comment by GiudGiud** [comment]
Hello

I dont think we have that option.
Can you try `build --serve --fast` to see if that s good enough? It still builds but without one of the slowest extensions.

Alternatively `--files` will let you specify which files to build, this should be a lot faster than the full documentation. This is mostly for writing new docs and checking that it works

Guillaume

**Comment by cticenhour** [comment]
You can definitely view the documentation site offline after building it once. When MooseDocs builds documentation, it places the files in a temporary location. This is, by default:

```
~/.local/share/moose/site
```

So, you could build the site and then look at it later by opening the `index.html` file in that location using your web browser. On a Mac, this is as easy as navigating to the location and then typing `open index.html` in a Terminal to open your default browser. The opened site is fully navigable. 

If you want to "install" those files to a particular location, we have a flag for that. Build MooseDocs by doing the following:

```
./moosedocs.py build --destination /my/wonderful/moose/local/website
``` 

Of course, you should replace `/my/wonderful/moose/local/website` with the location on your drive where you want to place the files.
